lib/checkPullRequestTemplate.js

```diff
@@ -162,7 +162,9 @@ const checkEssentialChecklist = (pullRequest) => {
         // Check if maintainer can actually modify the PR. This is because
         // it is possible for the maintainers to be able to modify the PR
         // but the PR author forgot to check it in the description.
-        if (!pullRequest.maintainer_can_modify) {
+        if (!pullRequest.maintainer_can_modify &&
+          !(pullRequest.head.repo.full_name ===
+            pullRequest.base.repo.full_name)){
```

Comment:
Can you extract this into a variable like ` isPrFromFork = pullRequest.head... === pullRequest.base...`?
Then use that with the `maintainer_can_modify` for the comparison.

Reply:
DONE